lisa/tools/lsvmbus.py

```diff
@@ -115,25 +137,42 @@ def _check_exists(self) -> bool:
             _exists = super()._check_exists()
             if not _exists:
                 self._command = previous_command
+
         return _exists
 
+    def _install_from_src(self) -> None:
+        wget_tool = self.node.tools[Wget]
+        file_path = wget_tool.get(self._lsvmbus_repo, "$HOME/.local/bin")
+        # make the download file executable
+        self.node.execute(f"chmod +x {file_path}")
+        self._command = file_path
+
     def install(self) -> bool:
-        linux_os: Linux = cast(Linux, self.node.os)
         package_name = ""
-        if isinstance(linux_os, Redhat):
+        if isinstance(self._linux_os, Redhat):
             package_name = "hyperv-tools"
-        elif isinstance(linux_os, Suse):
+            self._linux_os.install_packages(package_name)
+        elif isinstance(self._linux_os, Suse):
             package_name = "hyper-v"
-        elif isinstance(linux_os, Ubuntu) and not isinstance(linux_os, Debian):
+            self._linux_os.install_packages(package_name)
+        elif isinstance(self._linux_os, Ubuntu) and not isinstance(
```

Comment:
I may give incorrect inherits order of Debian and Ubuntu, please help on fixing it. After that, you can write Ubuntu here only.